cla_frontend/assets-src/javascripts/app/partials/case_detail.alternative_help.html

```diff
@@ -103,6 +103,12 @@ <h1 class="u-pullLeft">Alternative help</h1>
 
   <dir-pagination-controls on-page-change="pageChanged(newPageNumber)"></dir-pagination-controls>
 
+  <call-script>
```

Comment:
Ah. Yes you can. If you are in scope and have assigned a valid user, you can skip straight to alt-help.

I'll add some conditions.